saos-persistence/src/main/java/pl/edu/icm/saos/persistence/repository/JudgmentRepositoryCustomImpl.java

```diff
@@ -56,6 +60,29 @@
         return retJudgment;
         
     }
+    
+    @Override
+    @Transactional
+    public Map<Long, Long> countReferencingJudgmentsForNotIndexed() {
```

Comment:
After a little more thinking about it: maybe one service method: findAllNotIndexedJudgments returning List<JudgmentIndexing> would be the best solution. This way job reader will be very simple...